pkg/restic/repository_manager.go

```diff
@@ -278,3 +298,22 @@ func (rm *repositoryManager) exec(cmd *Command, backupLocation string) error {
 
 	return nil
 }
+
+// getInsecureSkipTLSVerifyFromBSL get insecureSkipTLSVerify flag from BSL configuration,
+// Then return --insecure-tls flag with boolean value as result.
+func GetInsecureSkipTLSVerifyFromBSL(backupLocation *velerov1api.BackupStorageLocation, logger logrus.FieldLogger) string {
```

Comment:
Maybe should not pass `logger` parameter for one basic function?

Reply:
Agree with your point to some extent. By now, there is only a nil and a not nil result, but before modification, this code is more complicated.
I am not sure whether there is a significant performance difference between passing in logger and not.
If not, I prefer to leave it as it's by now.